xstream/extensions/edgex_source.go

```diff
@@ -215,16 +215,15 @@ func (es *EdgexSource) getValue(r models.Reading, logger api.Logger) (interface{
 
 func (es *EdgexSource) getFloatValue(r models.Reading, logger api.Logger) (interface{}, error) {
 	if len(r.FloatEncoding) == 0 {
-		if strings.Contains(r.Value, "==") {
+		if strings.Contains(r.Value, "=") {
```

Comment:
Not sure if there are some cases that only include one '='?  I used same logic as in [this link](https://github.com/lenny-intel/app-service-examples/blob/refactor/app-services/advanced-filter-convert-publish/functions/convert.go#L60).

Reply:
pleae refer this post:
https://www.cnblogs.com/antineutrino/p/3756106.html

Reply:
ok, thanks for your info.